### DISCUSSION TITLE
Automatic manual mails send

### DISCUSSION THREAD

**Discussion by vovpff** [YetiForceCompany/YetiForceCRM#15517] [How to]
I has noticed what sometimes emails from system stuck in mail qeueue with some SMTP error. As I checked this is because my internet provoider has some lags and connection with SMTP lost durin emaiks sending.

Are there an way to send failed emails periodically (every 2 hours for example)?

**Comment by waran70** [comment]
smtp error can stop workflow running. xperienced that

**Reply by vovpff** [reply]
What you mean for that? I need to send failed mails queue what stopped with error in the past
